### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by aelkiss**
- update gems
- update mongo to mongodb 6.0.2
- remove unused prep_loadfiles.sh

### CODE REVIEW

File: spec/loader/cluster_loader_spec.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@
   end
 
   def count_clusters
-    Cluster.find_by(ocns: ocn).to_a.size
+    Cluster.for_ocns([ocn]).count
```
Comment (aelkiss) [inline spec/loader/cluster_loader_spec.rb:19]: `Cluster.find_by` returns a single cluster or nil; with the updated `mongoid` gem, `to_a` on a single cluster did not work. The change seems to better reflect the intention expressed in the method name.

**Comment by aelkiss** [comment]
@mwarin  Any concerns?

File: spec/loader/cluster_loader_spec.rb (hunk 1 above)
Comment (mwarin) [reply spec/loader/cluster_loader_spec.rb:19]: Do we gotta change:
```
clustering/cluster_holding.rb:51:        c = Cluster.find_by(ocns: holding.ocn)
reports/estimate.rb:31:        cluster = Cluster.find_by(ocns: ocn.to_i,
reports/etas_organization_overlap_report.rb:99:        cluster = Cluster.find_by(_id: c)
```

as well then?

**Review by mwarin — APPROVED** [review]
Looks fine, just one q that doesn't need to block merging.

File: spec/loader/cluster_loader_spec.rb (hunk 1 above)
Comment (aelkiss) [reply spec/loader/cluster_loader_spec.rb:19]: I checked for that as well, and I don't think so -- the problem was the `to_a`, not the `find_by` as such.

**Pull request merged**